server/src/main/java/org/apache/druid/segment/realtime/plumber/RealtimePlumberSchool.java

```diff
@@ -99,7 +101,9 @@ public RealtimePlumberSchool(
   public Plumber findPlumber(
       final DataSchema schema,
       final RealtimeTuningConfig config,
-      final FireDepartmentMetrics metrics
+      final FireDepartmentMetrics metrics,
+      final ServiceEmitter emitter,
```

Comment:
Look into instead just passing `ParseExceptionHandler` down instead of these two things....